src/components/Kv/KvRadio.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-	<span class="styled-radio">
+	<div class="styled-radio">
 		<input
 			type="radio"
-			value="radio-input"
+			:value="value"
```

Comment:
You could also remove this line and the prop, which will result in the `value` attribute being applied automatically by the `v-bind="$attrs"` line (the same things could be done for the `name` attribute)